packages/core/lib/commands/test/index.js

```diff
@@ -7,6 +7,11 @@ const command = {
       type: "boolean",
       default: false
     },
+    "console-log": {
```

Comment:
I know there's `truffle develop --log`, but maybe `truffle test --log` might be nice for this instead?

Alternatively, is this something to configure on a per-project basis? Did you consider adding it to the config?

Reply:
I think we want to make it so that we are sure the user deliberate wants us to link the logging library. Otherwise, it could incur additional cost and confusion.

Reply:
oh yeah I'm just talking about the name